From: Roland Rosenfeld <roland@debian.org>
Date: Fri, 15 Nov 2019 10:16:16 +0100
Subject: Generate the splash.xbm in a reproducible way, since the default
 pgmtopbm quantization method (boustrophedonic Floyd-Steinberg) generates
 different output on every build.

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -131,5 +131,6 @@ splash.xpm: $(top_srcdir)/splash-image/s
 	| sed '2 {s/static //; s/noname/spl_bckgnd_xpm/}' > $@
 
 splash.xbm: $(top_srcdir)/splash-image/spl_bckgnd.fig $(top_srcdir)/version.m4
-	fig2dev -L xbm -S 4 $(top_srcdir)/splash-image/spl_bckgnd.fig \
+	fig2dev -L ppm $(top_srcdir)/splash-image/spl_bckgnd.fig \
+	| ppmtopgm | pgmtopbm -d8 | pbmtoxbm \
 	| sed '1,3 s/noname/spl_bckgnd/; 3 s/char/unsigned &/' > $@
